concept/validation/schema-validation.feature

```diff
@@ -11,490 +11,734 @@ Feature: Schema validation
-    Given create attribute type: attr1, with value type: string
-    Given attribute(attr0) set abstract: true
-    Given attribute(attr1) set abstract: true
+    Given create attribute type: attr0
+    Given attribute(attr0) set value type: string
+    Given create attribute type: attr1
+    Given attribute(attr0) set annotation: @abstract
+    Given attribute(attr1) set annotation: @abstract
     Given attribute(attr1) set supertype: attr0
     Given create entity type: ent0
     Given create entity type: ent1
     Given entity(ent1) set supertype: ent0
     Given create relation type: rel0
-    Given relation(rel0) set relates role: role0
+    Given relation(rel0) create role: role0
     Given create relation type: rel1
-    Given relation(rel1) set relates role: role1
+    Given relation(rel1) create role: role1
     Given relation(rel1) set supertype: rel0
     Given transaction commits
 
-    When session opens transaction of type: write
-    Then attribute(attr1) set supertype: attr1; throws exception
+    When connection open schema transaction for database: typedb
+    Then attribute(attr1) set supertype: attr1; fails
 
-    When session opens transaction of type: write
-    Then attribute(attr0) set supertype: attr1; throws exception
+    When transaction closes
+    When connection open schema transaction for database: typedb
+    Then attribute(attr0) set supertype: attr1; fails
 
-    When session opens transaction of type: write
-    Then entity(ent1) set supertype: ent1; throws exception
+    When transaction closes
+    When connection open schema transaction for database: typedb
+    Then entity(ent1) set supertype: ent1; fails
 
-    When session opens transaction of type: write
-    Then entity(ent0) set supertype: ent1; throws exception
+    When transaction closes
+    When connection open schema transaction for database: typedb
+    Then entity(ent0) set supertype: ent1; fails
 
-    When session opens transaction of type: write
-    Then relation(rel1) set supertype: rel1; throws exception
+    When transaction closes
+    When connection open schema transaction for database: typedb
+    Then relation(rel1) set supertype: rel1; fails
 
-    When session opens transaction of type: write
-    Then relation(rel0) set supertype: rel1; throws exception
+    When transaction closes
+    When connection open schema transaction for database: typedb
+    Then relation(rel0) set supertype: rel1; fails
 
 
-  Scenario: An attribute-type must have the same value type as its ancestors
-    Given create attribute type: attr0s, with value type: string
-    Given attribute(attr0s) set abstract: true
-    Given create attribute type: attr0d, with value type: double
-    Given attribute(attr0d) set abstract: true
-    Given create attribute type: attr1, with value type: string
+  Scenario: An attribute-type must have the same value type as its ancestors, but not declare it redundantly
```

Comment:
Perfect attributetype.feature. We already have a similar test there, so it can even be just deleted after a recheck.